pkg/trustedcabundle/trustedcabundle.go

```diff
@@ -47,6 +48,10 @@ func HasCABundleAnnotationDisabled(ns client.Object) bool {
 // or update existing odh-trusted-ca-bundle configmap if already exists with new content of .data.odh-ca-bundle.crt
 // this is certificates for the cluster trusted CA Cert Bundle.
 func CreateOdhTrustedCABundleConfigMap(ctx context.Context, cli client.Client, namespace string, customCAData string) error {
+	// Adding newline breaker if user input does not have it
+	if !strings.HasSuffix(customCAData, "\n") {
+		customCAData += "\n"
+	}
```

Comment:
nit: maybe something we can do unconditionally, like 

```go
customCAData = strings.TrimSpace(customCAData) + "\n"
```

Reply:
thats nicer